code/mobs/mob.dm

```diff
@@ -73,12 +86,52 @@
 		return TRUE
 	src << "You do not have space for that."
 
+/mob/proc/equip(obj/item/equipable/I)
+	if(!can_equip(I))//already thinking about adding a text that tells you why you couldnt equip
+		return
+	//okay basic overlays working
+	src.overlays += create_overlay(I)
+	equipment[I.item_slot] = I
+	I.on_equip(src)
+	update_stats()
+	print_ac_list(equipment)
+	. = I.name
+
+/mob/proc/take_off(item_slot)
+	var/in_slot = equipment[item_slot]
+	if(!can_take_off(in_slot))
+		return
+	. = "[in_slot?:name] taken off"
```

Comment:
do not use `:`, make the type of `in_slot` `/obj/item/equipable`, also you don't need to get `name` here, it should just be `[in_slot]`